### ISSUE TITLE
My application has been suspended

### ISSUE THREAD

**Issue by nrbn** [manijak/nativescript-youtube-parser#5]
My application has been suspended because it violates the unauthorized  use of copyrighted content policy. Is it not allowed to use this plugin to play video from youtube? I had some issue before but not for copyrighted content.  It looks like it not allowed to get mp4 link of youtube videos. Here is more info:

[https://play.google.com/about/ip-deception-spam/impersonation-ip/unauthorized-copyright](url)

Should i use any youtube api key some place to get permission to show this videos?

**Comment by manijak** [comment]
This plugin is based on another npm plugin and is basically just a simple HTML parser. It opens a certain youtube page and tries to parse out the raw video url. No guarantee that it will work, and will probably stop working all together in the future. 

**I wouldn't recommend using this plugin in any production application. This could be treated as illegal by Google and YouTube,  and your application might be suspended.**

**No further development or maintenance will be done on this plugin.**

**Comment by johnny-stevie** [comment]
@manijak Since, this plugin cannot be used, could you suggest something else? Ive exhausted all my resources looking at nativescript-youtubeplayer, which is really buggy and unrealiable,webview, again,buggy and even at android-youtubeplayer, which is close to impossible to use in a template. All of which have turned out to be useless. Could you give some recommendations as to where to look next?

**Comment by manijak** [comment]
@johnny-stevie Yeah I understand that this is a bit of a hassle. To be honest I'm using this parser for iOS together with the `nativescript-videoplayer` plugin, until the parser stops working, and it probably will... 

For Android I use the following code to start a new activitiy for the video, and it will play the video either with the standard youtube player on the device or open the browser and play it there.

```js
try {
            var intent = new android.content.Intent(android.content.Intent.ACTION_VIEW, android.net.Uri.parse("vnd.youtube://" + selectedVideoItem.id.videoId));
            intent.setFlags(android.content.Intent.FLAG_ACTIVITY_NEW_TASK);
            application.android.foregroundActivity.startActivity(intent);
        }catch(e) {
            var secondaryIntent = new android.content.Intent(android.content.Intent.ACTION_VIEW, android.net.Uri.parse(selectedVideoItem.videoUrl));
            application.android.foregroundActivity.startActivity(secondaryIntent);
        }
```
